python/cugraph/cugraph/utilities/nx_factory.py

```diff
@@ -28,7 +29,82 @@
 nx = import_optional("networkx")
 
 
+def convert_unweighted_to_gdf(NX_G):
+    _edges = NX_G.edges(data=False)
+    src = [s for s, _ in _edges]
+    dst = [d for _, d in _edges]
```

Comment:
You can do this in one pass using `zip()`
```suggestion
    (src, dst) = zip(*_edges)
```

Reply:
I tried that and you can do it in one pass but the performance is not as good

Reply:
two pass takes 500ms,  the zip approach take 1.2 sec

Reply:
ah ok, I'll just mark as resolved then.